lib/validators/web_hook_validator.rb

```diff
@@ -0,0 +1,15 @@
+module Validators; end
+
+class Validators::WebHookValidator < ActiveModel::Validator
+
+  def validate(web_hook)
+    web_hook.errors.add(:payload_url, :blank, options) if web_hook.payload_url.blank?
```

Comment:
Is there a reason why we can't use a `validates :payload_url, presence: true` here?